fastqheat/config.py

```diff
@@ -1,4 +1,88 @@
+import logging
+import subprocess
+from configparser import ConfigParser
 from importlib.resources import files
+from pathlib import Path
+
+import click
+
+
+class FastQHeatConfigParser(ConfigParser):
+    def __init__(self, *, filename: str, click_param: click.Option):
+        super(FastQHeatConfigParser, self).__init__()
```

Comment:
Do you mean `super().__init__()`?

Reply:
it's the same

Reply:
`super(FastQHeatConfigParser, self).__init__()` executes `__init__()` of the `FastQHeatConfigParser` class

`super().__init__()` executes `__init__()` of the `ConfigParser` class

`super(ConfigParser, self).__init__()` executes `__init__()` of the `ConfigParser` class too, but it is Python's 2 syntax which people don't usually use in Python 3 (because there is no advantages of doing it the old and verbose way)